### PR TITLE
Updated Legion CLI and Model RestAPI documentations

### LINKED ISSUE #50: Update documentation about model REST api

### PR DESCRIPTION
**Pull request by aliaksandr-d**
This PR closes #50 and #51

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
@aliaksandr-d please build and provide link to view documentation

**Comment by aliaksandr-d** [comment]
@kirillmakhonin I've just rebuilt Legion, and here are fresh docs: https://cc.epm.kharlamov.biz/docs/0.5.6%2B180508150828.278.70cf9fc/

**Comment by aliaksandr-d** [comment]
@kirillmakhonin What do you mean, when say *remove legion pyserve* ? I'm not sure if it's a good idea, to remove *pyserve* from docs and leave it in code and tests and everywhere. Imho, if we don't need pyserve, we should create a new issue to remove it.

**Comment by kirillmakhonin** [comment]
@aliaksandr-d `pyserve` isn't a CLI tool, it's internal tool from model serving part, that cannot be used manually. Also that tool would be moved from legion package to toolchain package in close future.

**Comment by aliaksandr-d** [comment]
@kirillmakhonin i've removed *legion pyserve* docs. 
Anyway, to avoid desync between code and docs in future, we should add/update/remove functionality simultaneously with docs update. Every issue should have a small subtask *update docs if necessary*. It's much easier to update docs during code (or other stuff) change, but not in a month. 
**Same approach to Pull-requests, they should not be approved if there are no docs update.**

**Comment by kirillmakhonin** [comment]
@akharlamov, can you approve this PR, please?

File: legion/docs/source/command_utils.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Utils
```
Comment (akharlamov) [inline legion/docs/source/command_utils.md]: What is exactly the point of having command line help output from utilities put into documentation?

**Pull request closed without merging**